<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditBoard_AdjustElement_Blur</class>
 <widget class="QWidget" name="EditBoard_AdjustElement_Blur">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>62</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>590</width>
        <height>62</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_p2" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <property name="spacing">
             <number>2</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QToolButton" name="toolButton_11">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>38</width>
                <height>32</height>
               </size>
              </property>
              <property name="text">
               <string>CR</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="2">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>4</number>
            </property>
            <item>
             <widget class="QSpinBox" name="spinBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="focusPolicy">
               <enum>Qt::ClickFocus</enum>
              </property>
              <property name="suffix">
               <string/>
              </property>
              <property name="prefix">
               <string/>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>30</number>
              </property>
              <property name="value">
               <number>15</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="3">
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <property name="spacing">
             <number>4</number>
            </property>
            <item>
             <widget class="QToolButton" name="toolButton_6">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
              <property name="text">
               <string>^</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
